docs/reference/docs/reindex.asciidoc

```diff
@@ -1137,14 +960,211 @@ done
 ----------------------------------------------------------------
 // NOTCONSOLE
 
+[[docs-reindex-api-query-params]]
+==== {api-query-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=refresh]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=timeout]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=wait_for_active_shards]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=wait_for_completion]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=requests_per_second]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=scroll]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=slices]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=max_docs]
+
+[float]
+[[docs-reindex-api-response-body]]
+==== {api-response-body-title}
+
+//////////////////////////
```

Comment:
We typically don't include the response body snippet in this section:
https://github.com/elastic/docs/blame/master/shared/api-ref-ex.asciidoc#L116